etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiPartnerMetadataConverter.java

```diff
@@ -0,0 +1,86 @@
+package gov.hhs.cdc.trustedintermediary.external.hapi;
+
+import gov.hhs.cdc.trustedintermediary.etor.metadata.partner.PartnerMetadata;
+import gov.hhs.cdc.trustedintermediary.etor.metadata.partner.PartnerMetadataConverter;
+import gov.hhs.cdc.trustedintermediary.etor.operationoutcomes.FhirMetadata;
+import gov.hhs.cdc.trustedintermediary.etor.operationoutcomes.HapiFhirMetadata;
+import org.hl7.fhir.r4.model.OperationOutcome;
+
+public class HapiPartnerMetadataConverter implements PartnerMetadataConverter {
+
+    private static final HapiPartnerMetadataConverter INSTANCE = new HapiPartnerMetadataConverter();
+
+    public static HapiPartnerMetadataConverter getInstance() {
+        return INSTANCE;
+    }
+
+    private HapiPartnerMetadataConverter() {}
+
+    @Override
+    public FhirMetadata<?> extractPublicMetadataToOperationOutcome(
```

Comment:
While this was an existing method, I think it should get some apidoc

Reply:
I think the open api docs generate for the metadata endpoint. But also, I added docs to the class interface [here](https://github.com/CDCgov/trusted-intermediary/blob/d557068b91a8e6cc90916b782d1250e8fdeea39f/etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataConverter.java).